.github/workflows/call-run-individual-tests.yml

```diff
@@ -0,0 +1,86 @@
+name: Install deps and browsers, run tests, and upload artifacts
+
+on: workflow_call
+
+jobs:
+  # setup: 
+  #   name: Setup test environment
+  #   uses: ./.github/actions/setup-env.yml
+
+  test:
+    runs-on: ubuntu-latest
+    # strategy:
+    #   matrix:
+    #     os: [macos-latest, ubuntu-latest]
+    # runs-on: ${{ matrix.os }}
+
+    env:
+      ARCHIVE_EMAIL: ${{ secrets.ARCHIVE_EMAIL }}
+      ARCHIVE_PASSWORD: ${{ secrets.ARCHIVE_PASSWORD }}
+      PATRON_EMAIL: ${{ secrets.PATRON_EMAIL }}
+      PATRON_PASSWORD: ${{ secrets.PATRON_PASSWORD }}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+
+      - name: Run about tests
+        if: always()
```

Comment:
q: do you need the always?

Reply:
always can be removed, it's just added so the next step will still run even if the previous step has failed